### PR TITLE
Implementation of Kth prime factor in c++

### PR DESCRIPTION
**Pull request by Rishikeshrajrxl**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

(Yes)

### Checklist

- [X] I've read the contribution guidelines.
- [X] I've checked the issue list before deciding what to submit.
- [X] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

(Issues regarding #392 )

### CODE REVIEW

**Comment by Rishikeshrajrxl** [comment]
Waiting to get reviewed and Merged. So that I can work on further algorithm.

**Comment by HarshCasper** [comment]
@Rishikeshrajrxl Since you are a part of the Contributor's Hack and we have issued strict guidelines that the Contributions would start from 16th and you cannot submit a PR before someone assigns you that particular Issue. 

We will not be considering this PR for your Score in Contributor's Hack.

File: Miscellaneous/Problems/kth-prime factor.cpp [hunk 1]
```diff
@@ -0,0 +1,46 @@
+
+/* To find the  k-th prime factor of a given number
+
+//Example : n=12 ,  k=3
+  Output  : 3
+*/
+
+
```
Comment (HarshCasper) [inline Miscellaneous/Problems/kth-prime factor.cpp:8]: Remove Extra Spaces. They are redundant in the Code.

File: Miscellaneous/Problems/kth-prime factor.cpp [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+#include<iostream>
+
+int kPrimeFactor(int n, int k)
+{
+    while (n%2 == 0)
+    {
+        k--;
```
Comment (HarshCasper) [inline Miscellaneous/Problems/kth-prime factor.cpp:16]: Use meaningful variable names. Using `i` or `k` is not viable for anyone who is trying to understand your code.

File: Miscellaneous/Problems/kth-prime factor.cpp [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+    return -1;
+}
+
+
+int main()
+{
+    cout << kPrimeFactor(n, k) << endl;
+    return 0;
+}
+
```
Comment (HarshCasper) [inline Miscellaneous/Problems/kth-prime factor.cpp:45]: This Code is a complete copy of an already-existing [GeeksforGeeks Article](https://www.geeksforgeeks.org/k-th-prime-factor-given-number/) here. Trying to merge Copied Code into the Repository would be strictly dealt with and will lead to your block from this Repository.

**Review by HarshCasper — COMMENTED** [review]
I cannot merge this PR because your Code is a complete copy of an existing GeeksforGeeks Article. The main aim of this project is to bring all Data Structures and Algorithm under one umbrella but that does not involve copying code from other portals and merging them here.

**Comment by Rishikeshrajrxl** [comment]
Ok I will take care from next time

**Pull request closed without merging**